pkg/translate/translate.go

```diff
@@ -122,5 +122,10 @@ func (t *TranslateServiceServer) DownloadTranslationFile(
 		return nil, status.Errorf(codes.Internal, "serialize data: %s", err)
 	}
 
+	err = t.repo.LoadMessages(reqTranslationID)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "loading messages: %s", err)
+	}
```

Comment:
Load messages must return messages, and this code block should be before `to` function call, in this case instead of https://github.com/expect-digital/translate/blob/7a07d8440e7ad224edee58b10ab44125fa787653/pkg/translate/translate.go#L118